src/components/content/exercises/exercise.tsx

```diff
@@ -51,10 +49,16 @@ export function Exercise({ node, renderNested, path }: ExerciseProps) {
         href={node.href ? node.href : `/${node.context.id}`}
       />
 
-      {renderExerciseTask()}
-      {renderInteractive()}
+      <div className="flex justify-between">
+        {renderExerciseTask()}
```

Comment:
We need to wrap the exercise task in a div, because I have encountered this bug:

![grafik](https://user-images.githubusercontent.com/13507950/121055233-5f817c80-c7bd-11eb-9db9-b88351fb3740.png)